### PR TITLE
Added links to APIdocs where the levels are listed

### PR DESCRIPTION
**Pull request by antonpirker**
I updated the docs to include links to the autogenerated APIdocs that list the possible values for `level`.

Fixes: #5432

There is also a PR in the Python Repo where I updated the APIdocs, that will be released soon: https://github.com/getsentry/sentry-python/pull/2828

### CODE REVIEW

File: docs/platforms/python/integrations/django/http_errors.mdx [hunk 1]
```diff
@@ -27,4 +27,6 @@ def my_custom_page_not_found_view(*args, **kwargs):
 
 The error message you send to Sentry will have the usual request data attached.
 
+For details about `capture_message()` see the [API Documentation](https://getsentry.github.io/sentry-python/api.html#sentry_sdk.api.capture_message)
```
Comment (vivianyentran) [inline docs/platforms/python/integrations/django/http_errors.mdx:30]:
```suggestion
For details about `capture_message()`, see the [API Documentation](https://getsentry.github.io/sentry-python/api.html#sentry_sdk.api.capture_message).
```

File: docs/platforms/python/usage/set-level/index.mdx [hunk 2]
```diff
@@ -7,3 +7,5 @@ sidebar_order: 5
 The level - similar to logging levels - is generally added by default based on the integration. You can also override it within an event.
 
 <PlatformContent includePath="set-level" />
+
+For a list of possible levels see the [API Documentation](https://getsentry.github.io/sentry-python/api.html#sentry_sdk.api.set_level)
```
Comment (vivianyentran) [inline docs/platforms/python/usage/set-level/index.mdx:11]:
```suggestion
For a list of possible levels, see the [API Documentation](https://getsentry.github.io/sentry-python/api.html#sentry_sdk.api.set_level).
```

Comment (vivianyentran) [reply docs/platforms/python/usage/set-level/index.mdx:11]: Maybe I'm missing something, but I don't see a list of possible levels on that link?

**Review by vivianyentran — APPROVED** [review]

File: docs/platforms/python/usage/set-level/index.mdx (hunk 2 above)
Comment (antonpirker) [reply docs/platforms/python/usage/set-level/index.mdx:11]: Not yet, we need to update the APIdocs. The changes for that are already in the our pipeline and well be released soon.

**Pull request merged**